### PR TITLE
[ADD] [16.0] account_invoice_report_salesperson

### PR DESCRIPTION
**Pull request by Shide**
Adds Salesperson to Invoice reports

## Invoice report
image

## Invoice wihout Payments report
image

MT-6115 @moduon  @rafaelbn @yajo @EmilioPascual @fcvalgar please review if you want :)

### CODE REVIEW

**Review by fcvalgar — APPROVED** [review]
Functional review, 

LGTM, thank you @Shide 

image

image


image

File: account_invoice_report_salesperson/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+{
+    "name": "Account Invoice Report Salesperson",
+    "summary": "Salesperson info in Invoice report",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Account",
+    "website": "https://github.com/OCA/account-invoice-reporting",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["Shide"],
```
Comment (EmilioPascual) [inline account_invoice_report_salesperson/__manifest__.py:12]:
```suggestion
    "maintainers": ["Shide", "rafaelbn"],
```

**Review by EmilioPascual — CHANGES_REQUESTED** [review]

File: account_invoice_report_salesperson/views/report_invoice.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                name="salesperson"
+            >
+                <strong>Salesperson:</strong>
+                <p class="m-0" t-field="o.invoice_user_id.name" />
+                <p
+                    class="m-0 small text-muted"
+                    t-if="o.invoice_user_id.phone or o.invoice_user_id.mobile"
+                >
+                    <i class="fa fa-phone mr-1" aria-hidden="true" />
+                    <span t-esc="o.invoice_user_id.phone or o.invoice_user_id.mobile" />
```
Comment (yajo) [inline account_invoice_report_salesperson/views/report_invoice.xml]: this is [deprecated](https://github.com/odoo/odoo/blob/9ce5bc8881ae06b613ef61eb07453b224f62bae6/odoo/addons/base/models/ir_qweb.py#L1294):

```suggestion
                    <span t-out="o.invoice_user_id.phone or o.invoice_user_id.mobile" />
```

**Review by yajo — APPROVED** [review]

**Review by EmilioPascual — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**